clang/include/clang/DependencyScanning/DependencyScanningWorker.h

```diff
@@ -93,29 +93,39 @@ class DependencyScanningWorker {
 
   ~DependencyScanningWorker();
 
-  /// Run the dependency scanning tool for a given clang driver command-line,
-  /// and report the discovered dependencies to the provided consumer. If
-  /// TUBuffer is not nullopt, it is used as TU input for the dependency
-  /// scanning. Otherwise, the input should be included as part of the
-  /// command-line.
+  /// Run the dependency scanning tool for the given driver job command-line,
```

Comment:
Thanks for updating the documentation, this is great! Suggestion is to avoid confusion between the worker/tool and driver/frontend command lines.

```suggestion
  /// Run the dependency scanning worker for the given frontend command-line,
```